src/website/app/demos/ButtonGroup/examples/icons.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+import IconFlare from 'mineral-ui-icons/IconFlare';
+import IconSentimentSatisfied from 'mineral-ui-icons/IconSentimentSatisfied';
+import IconSentimentNeutral from 'mineral-ui-icons/IconSentimentNeutral';
+import IconSentimentDissatisfied from 'mineral-ui-icons/IconSentimentDissatisfied';
+import Button from '../../../../../library/Button';
+import ButtonGroup from '../../../../../library/ButtonGroup';
+import DemoLayout from '../../shared/DemoLayout';
+
+export default {
+  id: 'icons',
+  title: 'Icons',
+  description: `Buttons within ButtonGroup can contain [Icons](/components/icon),
+just as they do in [Button](/components/button#icons)`,
+  scope: {
+    Button,
+    ButtonGroup,
+    DemoLayout,
+    IconFlare,
+    IconSentimentDissatisfied,
+    IconSentimentNeutral,
+    IconSentimentSatisfied
+  },
+  source: `
+    <DemoLayout>
+      <ButtonGroup aria-label="Galaxies" mode="radio">
+        <Button defaultChecked iconStart={<IconFlare />}>Andromeda</Button>
+        <Button iconStart={<IconFlare />}>Triangulum</Button>
+        <Button iconStart={<IconFlare />}>Centaurus</Button>
+      </ButtonGroup>
+
+      <ButtonGroup aria-label="Sentiments" mode="radio">
```

Comment:
this is super fun. good example :)

Reply:
What he said. Great example!

Reply:
Agreed, you can thank @brentertz for that :)